kb/0002-scripting-overview.md

```diff
@@ -0,0 +1,46 @@
+# Scripting System Overview
+_[back to Index](../index.md)_
+
+![Storm Engine Logo](../media/SE_logo.png)
+
+**Related articles**: TBD
+
+Upon startup, the game (`ENGINE.exe`) will attempt to load any DLLs located in the `/modules/` directory. While these may contain any and all game logic, the game also reads non-compiled code located in `program` directory. These "scripts" are written in C using the game's API, and compiled at runtime using the built-in compiler. 
```

Comment:
`modules` folder was removed. now all libraries are linked statically with the main executable.

Reply:
Ah, I was looking at the Steam distro. On this note, could you please clarify what BIManager does? I'll include it in the description.

Reply:
I believe `BI` stands for `Battle Interface`